PG.StarWarsGame.Localisation/PG.StarWarsGame.Localisation/Attributes/DefaultLanguageAttribute.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Alamo Engine Tools and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+using System;
+using PG.StarWarsGame.Localisation.Languages;
+
+namespace PG.StarWarsGame.Localisation.Attributes;
+
+/// <summary>
+///     <see cref="Attribute" /> that can be used to mark a <see cref="IAlamoLanguageDefinition" /> as the default
+///     language.
+/// </summary>
+public class DefaultLanguageAttribute : Attribute
```

Comment:
make sealed, 
also this attribute should not inherit and only scope to classes|structs